cmd/proxy-client/main.go

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
I think we can move this application to a separate repo to prevent making monorepo